src/main/java/duke/command/AddCommand.java

```diff
@@ -0,0 +1,70 @@
+package duke.command;
+
+import duke.storage.Storage;
+//
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.TaskList;
+import duke.task.Todo;
+//
+import duke.ui.Ui;
+
+/**
+ * Represents the class to specify how to add command.
+ */
+public class AddCommand extends Command {
+    private String response;
+    private int type;
+
+    /**
+     * Adds the command.
+     *
+     * @param response The content of user input.
+     * @param type The type of adding command.
+     */
+    public AddCommand(String response, int type) {
+        this.response = response;
+        this.type = type;
+    }
+
+    /**
+     * Shows the task just be added.
+     *
+     * @param tasks The list of tasks.
+     * @param ui The user interaction instance.
+     * @param storage The instance to store data.
+     */
+    @Override
+    public void execute(TaskList tasks, Ui ui, Storage storage) {
```

Comment:
I think it would be good to specify what type corresponds to what action in the javadocs so that it will be easier to understand.